events/tests.py

```diff
@@ -1,3 +1,74 @@
 from .class_tests.event_tests import *  # noqa: F403 F401
 from .class_tests.participant_test import *  # noqa: F403 F401
 from .class_models.meeting_models import *  # noqa: F403 F401
+from .class_models.participant_model import EventParticipant
+from .class_models.event_models import Event, Colors
+from .forms import EventCreationForm
+from users import models
+from users.class_tests.users_tests import *  # noqa: F401 F403
+import pytest
+
+# form preferences -> new event preferences
+TITLE = 'new_form'
+LOCATION = 'new_form_location'
+DESCRIPTION = 'new_form_description'
+DATE_TIME_START = "11:11 PM 11-Nov-2011"
+DATE_TIME_END = "12:12 PM 12-Dec-2012"
+COLOR = Colors.BLACK
+
+
+@pytest.fixture
+def valid_event_data():
+    return {
+            "title": TITLE,
+            "location": LOCATION,
+            "description": DESCRIPTION,
+            "date_time_start": DATE_TIME_START,
+            "date_time_end": DATE_TIME_END,
+            "color": COLOR,
+        }
+
+
+@pytest.mark.django_db
+class TestCreateEventForm:
+
+    @pytest.mark.parametrize("invalid_data", [
+            # title cannot be None
+            {'title': None, 'date_time_start': DATE_TIME_START, 'date_time_end': DATE_TIME_END, 'color': COLOR},
+            # title cannot be blank
+            {'title': "", 'date_time_start': DATE_TIME_START, 'date_time_end': DATE_TIME_END, 'color': COLOR},
+            # date_time_start cannot be None
+            {'title': None, 'date_time_start': None, 'date_time_end': DATE_TIME_END, 'color': COLOR},
```

Comment:
here you want to check 
date_time_start can not be None
but you keep running with title = None
```suggestion
            {'title': TITLE, 'date_time_start': None, 'date_time_end': DATE_TIME_END, 'color': COLOR},
```

Reply:
done